docs/src/examples/components/Menu/Usages/index.tsx

```diff
@@ -24,6 +25,16 @@ const Usages = () => (
       description="When Submenu in MenuItem is controlled, then its 'menuOpen' prop value could be changed either by parent component, or by user actions (e.g. key press) - thus it is necessary to handle 'onMenuOpenChange' event."
       examplePath="components/Menu/Usages/MenuExampleWithSubmenuControlled"
     />
+    <ComponentExample
+      title="Menu with items containing tooltip"
+      description={
+        <>
+          {'The items inside the Menu as actionable element should be rendered with '}
```

Comment:
same here - are we advising client to always use `Tooltip` with any actionable element, or it is some different idea here?

Reply:
It is exactly that:

> The items inside the Menu as actionable element should be rendered with tooltip.


Can you suggest a better description?

Reply:
may suggest to embrace `as` in commas in both places, to proper indicate `because` semantics. This is similar to the following example, where `as` also makes semantic difference:

> I drove the car home as it was snowing. (i.e. `I was the car home as a very careful driver.`)
> I drove the car home, as it was snowing. (i.e. `I drove the car home because it was snowing.`)